src/api/services/usersService.ts

```diff
@@ -29,6 +29,7 @@ export const createUserInPortal = async (user: Omit<UserDTO, 'id' | 'acceptedTer
 export const isUserBelongsToParticipant = async (email: string, participantId: number) => {
   const user = await User.query()
     .where('email', email)
+    .where('deleted', 0)
```

Comment:
nit: this works, but I think using `.andWhere` is a bit clearer here, given it it used on the line below. Otherwise drop the `andWhere` on the below line, I'm pretty sure they behave the same.

Reply:
changed this to .andWhere()